src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -25,13 +25,14 @@
  * Various driver utilities.
  *
  */
-
 final class Util {
     final static String SYSTEM_SPEC_VERSION = System.getProperty("java.specification.version");
     final static char[] hexChars = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'};
     final static String WSIDNotAvailable = ""; // default string when WSID is not available
 
     final static String ActivityIdTraceProperty = "com.microsoft.sqlserver.jdbc.traceactivity";
+    static boolean activityIdTraceOn = "on"
```

Comment:
Static initialization of this variable will lock this to false, because the user hasn't provided the properties file yet. Even if they do, this variable won't be updated - we should leave isActivityTraceOn method as it is.